store/Makefile

```diff
@@ -78,12 +82,14 @@ push: check-registry
 	docker push $(MONITOR_DEV_IMAGE)
 
 set-image: check-deisctl
+	deisctl config store-admin set image=$(ADMIN_DEV_IMAGE)
 	deisctl config store-daemon set image=$(DAEMON_DEV_IMAGE)
 	deisctl config store-gateway set image=$(GATEWAY_DEV_IMAGE)
 	deisctl config store-metadata set image=$(METADATA_DEV_IMAGE)
 	deisctl config store-monitor set image=$(MONITOR_DEV_IMAGE)
 
 release:
+	docker push $(ADMIN_IMAGE)
```

Comment:
I'm ok with it being published to Docker Hub--we also have deis/test-etcd and deis/test-postgresql out there for convenience, and they're kind of optional containers.